app/models/atmosphere/virtual_machine_flavor.rb

```diff
@@ -60,6 +66,33 @@ class VirtualMachineFlavor < ActiveRecord::Base
 
     scope :active, -> { where(active: true) }
 
+    # Assumes only 1 vmf_osf for a specific os_family
+    def get_hourly_cost_for(os_family)
+      incarnation = flavor_os_families.find_by(os_family: os_family)
+      incarnation && incarnation.hourly_cost # nil if incarnation.blank?
```

Comment:
Is that really nil or simply false (the method call result) if incarnation.blank?